pkg/asset/installconfig/basedomain.go

```diff
@@ -21,6 +22,7 @@ import (
 
 type baseDomain struct {
 	BaseDomain string
+	Publish    types.PublishingStrategy
```

Comment:
Modifying this isn't sitting right with me. I'm going to lay out a different approach